tide_core.install

```diff
@@ -59,11 +59,15 @@ function tide_core_install() {
   // Update default Editorial workflow of Content Moderation.
   _tide_core_update_editorial_workflow();
 
+  $excluded = [
+    'tide_core_update_8015',
+    'tide_core_update_8018',
```

Comment:
Does this need to be excluded if you're fixing it below with the `$config->isNew()` check?